From 32717ef1c3337ec8a3d63cf753ea28415c7adc7d Mon Sep 17 00:00:00 2001
From: Ulises Mendez Martinez <ulises.mendez.martinez@intel.com>
Date: Tue, 12 Apr 2016 11:16:39 -0500
Subject: [PATCH 13/41] input: touch: silead: Configure maximum fingers number

Update to 10 the maximum number of fingers.

Change-Id: I5a326639ad6957d2a81915b252fb8bccfde42fe5
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-31785
Signed-off-by: Ulises Mendez Martinez <ulises.mendez.martinez@intel.com>
Reviewed-on: https://android.intel.com:443/493778
Reviewed-on: https://android.intel.com:443/504779
---
 drivers/input/touchscreen/silead.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/silead.c b/drivers/input/touchscreen/silead.c
index 23ec539..52198fe 100644
--- a/drivers/input/touchscreen/silead.c
+++ b/drivers/input/touchscreen/silead.c
@@ -76,7 +76,7 @@
 #define SILEAD_FW_NAME		"silead.fw"
 #define SILEAD_X_MAX		960
 #define SILEAD_Y_MAX		600
-#define SILEAD_MAX_FINGERS	5
+#define SILEAD_MAX_FINGERS	10
 #define SILEAD_PRESSURE		50
 #define SILEAD_FW_NAME_LEN	30
 
-- 
1.9.1

